owncloudData/src/test/java/com/owncloud/android/data/shares/datasources/OCRemoteShareesDataSourceTest.kt

```diff
@@ -20,40 +20,37 @@
 package com.owncloud.android.data.shares.datasources
 
 import com.owncloud.android.data.sharing.sharees.datasources.implementation.OCRemoteShareeDataSource
+import com.owncloud.android.data.sharing.sharees.datasources.mapper.RemoteShareeMapper
+import com.owncloud.android.domain.sharing.sharees.model.OCSharee
 import com.owncloud.android.lib.resources.shares.services.implementation.OCShareeService
-import com.owncloud.android.lib.resources.shares.GetRemoteShareesOperation
+import com.owncloud.android.domain.sharing.shares.model.ShareType
+import com.owncloud.android.lib.resources.shares.responses.ExactSharees
+import com.owncloud.android.lib.resources.shares.responses.ShareeItem
+import com.owncloud.android.lib.resources.shares.responses.ShareeOcsResponse
+import com.owncloud.android.lib.resources.shares.responses.ShareeValue
 import com.owncloud.android.utils.createRemoteOperationResultMock
 import io.mockk.every
 import io.mockk.mockk
-import org.json.JSONObject
 import org.junit.Assert.assertEquals
+import org.junit.Assert.assertFalse
 import org.junit.Assert.assertNotNull
+import org.junit.Assert.assertTrue
 import org.junit.Before
-import org.junit.Ignore
 import org.junit.Test
 
 // Needs to be instrumented since JSONObject is tied to the Android platform
 class OCRemoteShareesDataSourceTest {
```

Comment:
This test is inside shares package, i think it should be included in sharees package, what do you think?